src/components/messenger/list/conversation-list-panel/conversation-list-panel.scss

```diff
@@ -58,7 +58,7 @@ $side-padding: 16px;
       }
 
       .messages-list__tab-badge {
-        display: none;
+        display: block;
```

Comment:
@developerfred - You should be able to just remove this and it should display right?

Try removing 
```
      .messages-list__tab-badge {
        display: none;
      }
 ```